include/tools.h

```diff
@@ -167,5 +167,7 @@ std::vector<std::string> split(const std::string& str, const std::string& delims
  * @throw std::out_of_range if iso2 code is not known.
  */
 std::string converta2toa3(const std::string& a2code);
+
+std::string getFileContent(const std::string& path);
```

Comment:
Please add a doc string.
And making this function part of the public api could (should) be moved in a separated commit.

Reply:
@mgautierfr I will change the commit history once PR is ready to get merged